pontoon/api/tests/test_schema.py

```diff
@@ -327,12 +325,12 @@ def test_project_localizations_cyclic(client):
 
 
 @pytest.mark.django_db
-def test_locales_localizations_cyclic(client):
+def test_project_localizations_cyclic(client):
```

Comment:
Looks like the order of this test (and the one before) changed? No biggie, just noting that it makes the diff slightly larger.